lib/daru/view/adapters/googlecharts.rb

```diff
@@ -70,11 +71,13 @@ def show_in_iruby(plot)
         end
 
         def generate_html(plot)
-          # TODO: modify code
           path = File.expand_path('../../templates/googlecharts/chart_div.erb', __FILE__)
```

Comment:
I think instead of this `chart_div.erb` file, you can use your `static_html.erb` file. Since we have `initial_script` and  `chart_script ` (you are using it as `chart_div ` ).